### ISSUE TITLE
I have no idea of how to setup

### ISSUE THREAD

**Issue by AGPC94** [KoBeWi/Metroidvania-System#13]
When I put the addon in the folder, I don't know how to start to use it in a new project. Also I tested copying the example project and changing some things, but I got weird errors.

So, could you mind to make a guide to know how to start a new project with your asset? It would be a great help. Thank you.

**Comment by KoBeWi** [comment]
I plan on making a clean start guide, as some people have requested it.

What errors are you getting from example project? Note that you need its `project.godot` to run it, because it contains some settings related to the plugin.

**Comment by KoBeWi** [comment]
In 8a1fbe9dca2efbc11fa3cc48a3fd93f4a4f2b7bc I added a quick start guide that explains some basics. Check in the README if that's enough to get started.

Although I also found some pitfalls that I'm going to iron out.
- changing map base directory requires editor restart, otherwise there are some problems in the editor
- the base directory does not have `/` when selected, which breaks the scene changing in sample project
- scene transitions have one step that's rather difficult to figure out, maybe it can be simplified
